[QUAL-1341] Product Updates June 2024 (#216)
andela-technology/qualified-docs: content/changes.md

Learn about notable new features, fixes, and other improvements to the application!

## June 30, 2024

### Bug Fixes 

- **Billing Page Displaying Wrong Number of Active Users**
 

>   The blind reviewers were not counting as restricted seats generating inconsistencies in the total number of active users.

Reviewer: ```suggestion
  Blind reviewers were not being counted as restricted seats, generating inconsistencies in the total number of active users.
```

Learn about notable new features, fixes, and other improvements to the application!

## June 30, 2024

### Bug Fixes 

> - **Demo Data - Scoring Inaccurate** 
> 
>   We disabled the option to generate demo data on new teams in order to keep our demo data consistent for all our customers. 
>  

Reviewer: ```suggestion
```

This isn't fixed yet, so it's too soon to alert customers of our rather embarrassing temporary workaround. Once we do fix this, though, this will be a good thing to include.

- **Billing Page Displaying Wrong Number of Active Users**
 
  The blind reviewers were not counting as restricted seats generating inconsistencies in the total number of active users.

> - **'See all of the programming languages we support' link is broken**
>  
>   In our website the ‘See all of the programming languages we support was broken it went nowhere and now it’s fixed redirecting this [page](https://www.qualified.io/features#languages)
> 

Reviewer: ```suggestion
```

Too trivial to mention and applies more to prospective customers visiting our landing page for the first time rather than existing customers using the actual app.
